src/main/java/io/cdap/cdap/metrics/process/gcp/MetricsMapping.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.metrics.process.gcp;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+
+/**
+ * Google Cloud monitoring metrics definition.
```

Comment:
nit: give more details on what mappings are, map from what to what?

Reply:
If you can give an example of the mapping, it would be great. Just reading this class, I have a hard time to understand what the mapping source and destination.

Reply:
Added examples for all config classes.